### DISCUSSION TITLE
I want to a recipe to turn a 3d molecule to a actual compound in lab in a computational way

### DISCUSSION THREAD

**Discussion by biohacker0** [rdkit/rdkit#6775] [FAQ]
So I have some 3d molecules, now what I want is a recipe on like what precursors would I need , what temp, pressure etc. 
Is there a computational way to find this out, so that I have a like a recipe book to just follow in lab and I get to synthesize my molecule to a actual compound substance that I can test.

why -> so I have 32 3d stereoisomer molecules of estradiol , now what I want is a way to synthesize them in lab, so I want a sort of step by step guide that gives me details of all compound and their alternatives that can be used , like the whole process.

does anyone has any info related to any computational way for that.

**Comment by kienerj** [comment]
You could use the computational power of an organic chemists brain.  And I mean it not only as a stupid joke but really mean it.

This is called retro synthesis and there are paid and free tools available with different chances of being successful. Still most tools require an organic chemist to judge the suggestions and then define what you call "receipt". No tool will give you out a receipt that will just work.

[IBM](https://rxn.res.ibm.com/) did some work into this, as a hint to get you started. but again you will need someone to judge the output in terms of it it works at all and if it might be dangerous. So organic chemistry synthesis expertise is simply a must-have.

**Reply by biohacker0** [reply]
oh thanks for info , I looked into the IBM's: [https://rxn.res.ibm.com](https://t.co/AZlaIcUcJR) , and there are indeed some retrosynthesis software's, although none of them give exact steps , more like helpers for organic chemists.

I guess someone needs to scrape all data from PubChem, ChemSpider, and just train models to build a LLM for chemistry.

sort of like alpha fold for organic chemistry.
